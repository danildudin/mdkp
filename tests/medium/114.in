30 7 0
33 107 795 547 788 808 275 219 284 798 739 975 131 616 251 2 380 908 908 85 113 726 286 679 787 324 154 917 655 748 
42 954 628 839 704 963 127 879 883 941 259 577 819 166 410 31 987 989 632 179 480 959 137 717 954 272 729 172 414 798 
369 961 151 666 984 301 467 667 751 690 898 308 731 3 495 688 373 978 527 346 864 320 202 549 305 182 971 570 542 155 
357 520 846 427 999 894 185 402 173 97 638 505 136 715 353 188 766 412 780 216 447 618 13 747 71 906 120 998 838 373 
733 442 824 683 326 185 300 961 235 294 352 562 802 84 361 806 638 826 357 259 162 636 828 808 278 350 101 342 530 368 
168 406 271 963 442 428 271 506 902 945 38 170 711 865 770 716 86 380 701 118 641 152 136 791 261 994 571 983 528 945 
580 502 708 438 738 152 950 457 594 27 971 871 37 220 566 254 274 151 250 21 571 434 447 657 975 480 512 196 932 758 
448 787 584 912 885 119 929 338 30 541 263 814 840 746 745 246 895 157 282 32 530 233 753 594 147 382 231 31 998 248 
6562 9818 8924 5381 6161 9705 8216 
