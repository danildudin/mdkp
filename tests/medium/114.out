9297
13
2 4 5 6 10 11 13 18 22 24 27 28 29 
